guided-match/dml-versions/v024.sql

```diff
@@ -0,0 +1,196 @@
+('Waste Bin','Furniture and Associated Services', 'Buy a range of furniture for use in public sector organisations, as well as repair and renovation services.'),
+('Witness Screens','Furniture and Associated Services', 'Buy a range of furniture for use in public sector organisations, as well as repair and renovation services.'),
+('Witness Stand','Furniture and Associated Services', 'Buy a range of furniture for use in public sector organisations, as well as repair and renovation services.'),
+('Workstation','Furniture and Associated Services', 'Buy a range of furniture for use in public sector organisations, as well as repair and renovation services.');
+
+insert into search_terms (search_term)
+select  distinct lower(search_term)
+from    temp_domain_values td;
+-----------------------
+
+-- MAPPING NEW KEYWORDS WITH GM LITE
+insert into search_domains (search_id,journey_id,modifier_journey_name,journey_selection_text,journey_selection_description)
+select distinct
+		st.search_id,
+		CAST('c9dd4455-7d23-4822-9912-eab4da9fc5a2' AS uuid) as journey_id,
+		'GM Lite' as modifier_journey_name,
+		'Something else' as journey_selection_text,
+		'What I need is not listed here' as journey_selection_description
+from   search_terms st
+join   temp_domain_values td on lower(td.search_term) = st.search_term;
+------------------------------------
+
+-- MAPPING OLD KEYWORDS WITH THE NEW JOURNEY
+insert into temp_domain_values VALUES
+('Bed','Furniture and Associated Services', 'Buy a range of furniture for use in public sector organisations, as well as repair and renovation services.'),
+('Chair','Furniture and Associated Services', 'Buy a range of furniture for use in public sector organisations, as well as repair and renovation services.'),
+('Desk','Furniture and Associated Services', 'Buy a range of furniture for use in public sector organisations, as well as repair and renovation services.'),
+('Furniture','Furniture and Associated Services', 'Buy a range of furniture for use in public sector organisations, as well as repair and renovation services.'),
+('Storage','Furniture and Associated Services', 'Buy a range of furniture for use in public sector organisations, as well as repair and renovation services.');
+
+insert into search_domains (search_id,journey_id,modifier_journey_name,journey_selection_text,journey_selection_description)
+select distinct st.search_id,
+       CAST('085df7f9-8014-4350-b3a4-b1eadf5c3766' AS uuid) as journey_id,
+	   td.domain_name,
+	   td.domain_name,
+	   td.domain_desc
+from   search_terms st
+join   temp_domain_values td on lower(td.search_term) = st.search_term;
+--------------------------------------------
+
+INSERT INTO error_messages (error_message_code, error_summary, error_message)
+VALUES
+('ERR_GM066','Select the product you need','Select the product you need'),
```

Comment:
This is the same error message as ERR_GM038 so we can remove this line as there is no need of 2 error entry of the same message